src/components/AuthForm.js

```diff
@@ -1,7 +1,38 @@
 import React from "react";
 import InputForm from "./Form";
+import {useState} from "react";
+import axios from 'axios';
+
+//let axios = require('axios');
 
 function AuthForm (props) {
+  const [username, setUsername] = useState("");
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  async function submitData () {
+    const data = {
+      "user": {
+      username,
+      email,
+      password
+      }
+    }
+
+    let register = await axios.post('https://launchup-prisma.herokuapp.com/api/users', data)
+    let tokenInfo = sessionStorage.setItem('jwt', register.data.user.token);
+  }
+
+  function onHandleChange (data) {
+    if (data.name === "Username") {
```

Comment:
Lets make this into a switch statement so its easier to read and maintain.

Reply:
This has been updated. Thanks!